sphinx/builders/latex.py

```diff
@@ -211,6 +211,11 @@ def finish(self):
             logger.info(bold('copying images...'), nonl=1)
             for src, dest in iteritems(self.images):
                 logger.info(' ' + src, nonl=1)
+                subdir, _filename = path.split(dest)
```

Comment:
Fixed, now I replace `/` by `__slash__` (without checking if that new file name exists, I assume the risk is low enough not to complicate the code further)